rel/files/mongooseim

```diff
@@ -157,7 +164,13 @@ case "$1" in
         logger -t "$SCRIPT[$$]" "Starting up"
 
         # Start the VM
-        exec_echo "$BINDIR/erlexec" -boot "$RUNNER_BASE_DIR/releases/$APP_VSN/$BOOTFILE" -embedded -config "$RUNNER_ETC_DIR/app.config" -args_file "$RUNNER_ETC_DIR/vm.args" -args_file "$RUNNER_ETC_DIR/vm.dist.args" -- ${1+"$@"}
+        exec_echo "$BINDIR/erlexec" $FOREGROUNDOPTIONS \
```

Comment:
Something wrong with embedded argument. 
Wondering, if erl wants the argument before any config argument. Will look deeper on Monday.

Reply:
This exact commit is needed to fix CI
https://github.com/arcusfelis/MongooseIM/pull/33/commits/58ee78fafba2474a421ba5478533b93b3144faa4

Reply:
So, cherry-pick the commit or apply the changes, push, let's wait for CI to pass on the updated build.
Once it's ready - we can merge it.